crates/pixi_uv_conversions/src/requirements.rs

```diff
@@ -105,23 +106,36 @@ pub fn as_uv_req(
                 },
             ..
         } => RequirementSource::Git {
-            repository: RepositoryUrl::parse(git.as_str())
-                .map_err(|err| AsPep508Error::UrlParseError {
-                    source: err,
-                    url: git.to_string(),
-                })?
-                .into(),
+            // Url is already a git url, should look like:
+            // - 'ssh://git@github.com/user/repo'
+            // - 'https://github.com/user/repo'
+            repository: {
+                if git.scheme().strip_prefix("git+").is_some() {
+                    // Setting the scheme might fail, so using string manipulation instead
+                    let url_str = git.to_string();
+                    let stripped = url_str.strip_prefix("git+").unwrap_or(&url_str);
+                    // Reparse the url with the new scheme, should be safe but could fail so fall back to the original.
+                    Url::parse(stripped).unwrap_or(git.clone())
```

Comment:
Should this not be an actual error? When would it be okay if this fails?

Reply:
I didn't so it's not an error